src/packages/README.md

```diff
@@ -9,17 +9,90 @@ They have different levels of completeness, maturity, security, and support.
+
+## Package managers in CheriBSD
+
+*Note:* As of this writing we only provide packages for Morello systems.
+We aim to add CHERI-RISC-V package sets in the near future.
+
+CheriBSD includes two package managers: `pkg64` for hybrid ABI packages and
+`pkg64c` for CheriABI packages. The FreeBSD package manager `pkg` is not
+available on CheriBSD. The `pkg` program was replaced with a script that prints
+information on available package managers and always returns the exit code 1
+making `pkg` commands invalid on CheriBSD:
+```
+# pkg
+Please use the ABI-specific pkg variant:
+* pkg64c for experimental software built for CheriABI;
+* pkg64 for stable software built for the hybrid ABI.
+
+Replace pkg with pkg64c or pkg64 in your command and try again.
+# echo $?
+1
+```
+
+We expect that `pkg64c` will be renamed to `pkg` in a future CheriBSD release.
+The intention is that, over time, the CheriABI packages will become more
+mature, and hence the preferred collection for day-to-day use.
+
+The syntax of the `pkg64` and `pkg64c` commands match the syntax of the `pkg`
+command from FreeBSD.
+You can find information on package manager commands in FreeBSD's `pkg` manual
+pages shipped with CheriBSD, e.g. for the commands `pkg64 rquery` and
+`pkg64c rquery`, execute `man pkg-rquery`.
+
+Additionally, you can learn more about FreeBSD's package manager by reading the
+[pkg(8) manual page](https://www.freebsd.org/cgi/man.cgi?pkg(7)) online
+and the [FreeBSD Handbook chapter on packages and
+ports](https://docs.freebsd.org/en/books/handbook/ports/).
+
 ## Hybrid ABI packages
 
 **Hybrid ABI packages** are compiled almost identically to packages in the
 baseline non-CHERI architecture (e.g., Armv8-a for Morello, and 64-bit
 RISC-V for RISC-V), and do not have improvements in memory protection or
 software compartmentalization.
-There are over 20,000 packages available.
-These are considered *appropriate for day-to-day use*.
 
-These packages are installed in the `/usr/local64` hierarchy. By default
-`/usr/local64/bin` and `/usr/local64/sbin` should be in your path.
+There are currently over 20,000 hybrid ABI packages available, including:
+* Compilers:
+  * `llvm-morello`;
+
+    Includes Clang (a C/C++/Objective-C compiler), LLD (a linker), and the LLVM
+    infrastructure for the Arm Morello architecture.
+    Binaries installed with this package have the suffix `-morello`.
+
+  * `llvm`;
+
+    Adds links in `/usr/local64/bin` to allow the `llvm-morello` package to be
+    used as the default LLVM package, without the suffix `-morello`,
+    e.g. `clang` instead of `clang-morello`.
+
+  * `llvm-base`;
+
+    Adds links in `/usr/bin` to allow the `llvm` package to be used in place of
+    a base-system toolchain. The `cc` script installed with this package adds
+    compiler flags required to natively compile code for a CHERI-enabled
+    architecture.
+* Debuggers: `gdb-cheri`;
+* Editors: `nano`, `vim`;
```

Comment:
Maybe add `emacs-nox`

Reply:
We don't have a package with `emacs-nox` yet. It doesn't build:
https://poudriere.cheribsd.org/data/cheribsd-aarch64-main/latest-per-pkg/emacs-nox-28.1_1%2C3.log
https://poudriere.cheribsd.org/data/cheribsd-aarch64-main/latest-per-pkg/emacs-nox-27.2_1%2C3.log

Reply:
Yes, it's located at `/zdata/cheri/output/rootfs-aarch64/usr/include`, alongside `execinfo.h` for which the error is reported. I'll have a look at this issue after the GUI stack packages.